docs/Documentation/Development/Languages/Python/KestrelParallelPytonJupyter/README.md

```diff
@@ -0,0 +1,192 @@
+# Interactive Parallel Python on Kestrel with Jupyter
+
+For a general introduction to using Jupyter notebooks on Kestrel, please refer to the [official documentation](https://nrel.github.io/HPC/Documentation/Development/Jupyter/). For quick low compute intensive jobs, please visit the [Kestrel JupyterHub](https://kestrel-jhub.hpc.nrel.gov/). This page conveys specific information on how to leverage parallel computing on NREL HPC systems with python through Jupyter notebooks. Accompanying notebooks to test if the environment and the allocation has been configured correctly are in the 'exampleNotebooks' folder.
```

Comment:
```suggestion
For a general introduction to using Jupyter notebooks on Kestrel, please refer to the [official documentation](https://nrel.github.io/HPC/Documentation/Development/Jupyter/). This page covers how to leverage parallel computing with python through Jupyter notebooks on compute nodes. Accompanying notebooks to test if the environment and the job have been configured correctly are in the 'exampleNotebooks' folder.
```

Reply:
kestrel jupyterhub is covered on the documentation page that you linked to and isn't really relevant to this page so I don't think it needs to be mentioned here to make the paragraph more concise.